apps/src/templates/rubrics/teacherRubricRedux.ts

```diff
@@ -49,6 +77,21 @@ const teacherRubricReduxSlice = createSlice({
     setUserHasTeacherFeedback(state, action: PayloadAction<number>) {
       state.hasTeacherFeedbackMap[action.payload] = true;
     },
+    setAiEvalStatusCounters(
+      state,
+      action: PayloadAction<AiEvalStatusCounters>
+    ) {
+      state.aiEvalStatusCounters = action.payload;
+    },
+    setAiEvalStatusMap(state, action: PayloadAction<AiEvalStatusMap>) {
+      state.aiEvalStatusMap = action.payload;
+    },
+    setUserAiEvalStatus: (
+      state,
+      action: PayloadAction<{userId: number; status: string}>
```

Comment:
This works and you can use `setUserAiEvalStatus({userId: userId, status: status})`, another option is to add a prepare statement like [this](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/teacherDashboard/teacherSectionsRedux.ts#L368) example and then you can   use it without the object types in the function call (e.g. `setUserAiEvalStatus(userId, status)`).